src/trio/_core/_run.py

```diff
@@ -1023,6 +1023,7 @@ async def __aexit__(
         exc: BaseException | None,
         tb: TracebackType | None,
     ) -> bool:
+        del tb
```

Comment:
This seems intimidating, when you're adding a test could you add another to make sure no stackframes are getting lost? (I assume they're part of `exc` in addition?)

Reply:
The tb is never needed in a context manager, and cpython are looking to deprecate the three arg form in favor of a one arg exc_value only form